api/_mixins/LinkStyle__ProcessingInstruction.json

```diff
@@ -0,0 +1,55 @@
+{
+  "api": {
+    "ProcessingInstruction": {
+      "sheet": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ProcessingInstruction/sheet",
+          "spec_url": "https://drafts.csswg.org/cssom/#the-linkstyle-interface",
+          "support": {
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": "18"
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": "9"
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "14"
+            },
+            "safari": {
+              "version_added": "3"
```

Comment:
Is this based on testing in Safari 2+3? I wonder because it seems a little bit too perfect that `sheet` was introduced in the same Safari version on all three interfaces. Often things have been a bit messier than that, like in the Opera case. (When it's all in the very first release of a browser, that's not as suspicious to me.)

Reply:
It was a guesstimate based upon the other interfaces, I wasn't able to get the `ProcessingInstruction` test working in Safari.  I just took a closer look at this, however, and it appears that the commit history suggests it was probably around since Safari 1: https://trac.webkit.org/browser/webkit/trunk/WebCore/khtml/ecma/kjs_dom.cpp?annotate=blame&rev=3376#L1160

Reply:
Yep, in fact it's present in all three APIs in Safari 1.  I don't know why I thought it was Safari 3 at first...

Reply:
Maybe the 3+1 pairing of many SVG APIs was fresh in your memory?